packages/ui-components/src/components/tag/tag.tsx

```diff
@@ -0,0 +1,31 @@
+import { Component, Host, Prop, h } from '@stencil/core';
+
+@Component({
+	tag: 'kv-tag',
+	styleUrl: 'tag.scss',
+	shadow: true
+})
+export class KvTag {
+	/** Tag text */
+	@Prop({ reflect: true }) text: string;
+	/** (optional) Determines if the Tag has content on its left slot */
+	@Prop({ reflect: true }) hasLeftSlot?: boolean = false;
```

Comment:
You don't need this prop you can know that with 
```
private get hasLeftSlot() {
	return !isNil(this.el.querySelector('[slot="left-slot"]'));
}
```
and add a `left-slot` name to the slot

Reply:
Thanks! Will update